### ISSUE TITLE
Entering Settings causes error in devtools

### ISSUE THREAD

**Issue by BOHEUS** [twentyhq/twenty#6816]
**Scenario:**
1. Log in
2. Go to Settings

**Actual:** Upon entering Settings, errors appear
[Screencast from 2024-08-31 12-55-20.webm](https://github.com/user-attachments/assets/022f2e04-30e2-487f-be50-0b31fc9e5b11)

**Expected:** Going to Settings shouldn't cause the problems

Error:
```
Warning: React does not recognize the `data-testId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `data-testid` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
input
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-2IPCHOKD.js?v=4a09cb80:1278:45
TextInputV2Component@http://localhost:3001/src/modules/ui/input/components/TextInputV2.tsx:106:30
TextInput@http://localhost:3001/src/modules/ui/input/components/TextInput.tsx:18:26
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
NameFields@http://localhost:3001/src/modules/settings/profile/components/NameFields.tsx:29:27
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
Section@http://localhost:3001/src/modules/ui/layout/section/components/Section.tsx:28:24
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
ScrollWrapper@http://localhost:3001/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx:29:30
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
SettingsPageContainer@http://localhost:3001/src/modules/settings/components/SettingsPageContainer.tsx:43:38
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
PagePanel@http://localhost:3001/src/modules/ui/layout/page/PagePanel.tsx:22:26
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
PageBody@http://localhost:3001/src/modules/ui/layout/page/PageBody.tsx:40:25
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
SubMenuTopBarContainer@http://localhost:3001/src/modules/ui/layout/page/SubMenuTopBarContainer.tsx:24:39
SettingsProfile
RenderedRoute@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4038:7
Routes@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4476:7
Suspense
SettingsRoutes@http://localhost:3001/src/SettingsRoutes.tsx:142:31
RenderedRoute@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4038:7
Outlet@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4410:20
ErrorBoundary@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-error-boundary.js?v=4a09cb80:18:5
AppErrorBoundary@http://localhost:3001/src/modules/error-handler/components/AppErrorBoundary.tsx:13:33
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
div
MotionComponent@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-5IHKZV2V.js?v=4a09cb80:277:40
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
div
withEmotionCache2/<@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1278:45
DefaultLayout@http://localhost:3001/src/modules/ui/layout/page/DefaultLayout.tsx:72:22
RenderedRoute@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4038:7
Outlet@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4410:20
DialogManager@http://localhost:3001/src/modules/ui/feedback/dialog-manager/components/DialogManager.tsx:15:30
DialogManagerScope@http://localhost:3001/src/modules/ui/feedback/dialog-manager/scopes/DialogManagerScope.tsx:11:35
SnackBarProvider@http://localhost:3001/src/modules/ui/feedback/snack-bar-manager/components/SnackBarProvider.tsx:34:33
ThemeContextProvider@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/packages/twenty-ui/src/theme/provider/ThemeContextProvider.tsx:12:37
ThemeProvider2@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/chunk-3IPCHOKD.js?v=4a09cb80:1308:22
AppThemeProvider@http://localhost:3001/src/modules/ui/theme/components/AppThemeProvider.tsx:16:33
PrefetchDataProvider@http://localhost:3001/src/modules/prefetch/components/PrefetchDataProvider.tsx:12:37
RelationPickerScope@http://localhost:3001/src/modules/object-record/relation-picker/scopes/RelationPickerScope.tsx:11:36
PreComputedChipGeneratorsProvider@http://localhost:3001/src/modules/object-metadata/components/PreComputedChipGeneratorsProvider.tsx:16:50
ObjectMetadataItemsProvider@http://localhost:3001/src/modules/object-metadata/components/ObjectMetadataItemsProvider.tsx:18:44
ApolloMetadataClientProvider@http://localhost:3001/src/modules/object-metadata/components/ApolloMetadataClientProvider.tsx:14:45
AuthProvider@http://localhost:3001/src/modules/auth/components/AuthProvider.tsx:15:29
UserProvider@http://localhost:3001/src/modules/users/components/UserProvider.tsx:19:29
ChromeExtensionSidecarProvider@http://localhost:3001/src/modules/chrome-extension-sidecar/components/ChromeExtensionSidecarProvider.tsx:53:47
ClientConfigProvider@http://localhost:3001/src/modules/client-config/components/ClientConfigProvider.tsx:13:37
ApolloProvider@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/@apollo_client.js?v=4a09cb80:161:16
ApolloProvider@http://localhost:3001/src/modules/apollo/components/ApolloProvider.tsx:13:31
ProvidersThatNeedRouterContext@http://localhost:3001/src/App.tsx:60:26
RenderedRoute@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4038:7
RenderErrorBoundary@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:3995:5
DataRoutes@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:5157:7
Router@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4424:7
RouterProvider@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-router-dom.js?v=4a09cb80:4972:7
App@http://localhost:3001/src/App.tsx:465:35
r2@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-helmet-async.js?v=4a09cb80:313:21
ExceptionHandlerProvider@http://localhost:3001/src/modules/error-handler/components/ExceptionHandlerProvider.tsx:11:41
IconsProvider@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/packages/twenty-ui/src/display/icon/providers/IconsProvider.tsx:13:30
SnackBarProviderScope@http://localhost:3001/src/modules/ui/feedback/snack-bar-manager/scopes/SnackBarProviderScope.tsx:11:38
CaptchaProvider@http://localhost:3001/src/modules/captcha/components/CaptchaProvider.tsx:12:32
ErrorBoundary@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/react-error-boundary.js?v=4a09cb80:18:5
AppErrorBoundary@http://localhost:3001/src/modules/error-handler/components/AppErrorBoundary.tsx:13:33
RecoilRoot_INTERNAL@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/recoil.js?v=4a09cb80:2794:29
RecoilRoot@http://localhost:3001/@fs/home/user/Desktop/projects/twenty/node_modules/.vite/packages/twenty-front/deps/recoil.js?v=4a09cb80:2915:7
```

**Comment by BOHEUS** [comment]
Related to #6733

**Comment by charlesBochet** [comment]
Indeed, I think I was the one introducing this issue :)

Tagging this as good first issue

**Comment by charlesBochet** [comment]
Indeed, I think I was the one introducing this issue :)

Tagging this as good first issue

**Comment by DurgaManikantaSai** [comment]
seems like can be done easily, I want to work on it. May I ? @charlesBochet .

**Comment by BOHEUS** [comment]
@DurgaManikantaSai it's already done in linked PR, if you want to contribute, take another good first issue

**Comment by FelixMalfait** [comment]
Closing the issue to avoid confusion as it's resolved in #6736 by @Nabhag8848 but I can't assign him since he did not comment on the issue. Thanks a lot @Nabhag8848!

**Comment by FelixMalfait** [comment]
https://github.com/twentyhq/twenty/pull/6736/files#diff-a48e37b645699a3309d359e3084f0218c38441585d8e3884083c411dacb096f4R184
